pkg/component/autoscaling/vpa/crd.go

```diff
@@ -20,23 +23,36 @@ var (
 	verticalPodAutoscalerCheckpointCRD string
 
 	crdResources map[string]string
+	// resourceObjectKeys is a slice of the CRD object keys
+	resourceObjectKeys []client.ObjectKey
```

Comment:
```suggestion
	crdNames []string
```

No doc string required if the var name is clear :)

Reply:
I would still prefer keeping it a `client.ObjectKey` slice to prevent unmarshalling the resources on every `Wait()`